.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Description
+
+<!--
+Thank you for your pull request (PR)!
+
+Please provide a description of what your PR does providing a link (if applicable) to the issue it fixes.
+-->
+Fixes # (issue)
+
+## Type of change
+
+<!---
+What types of changes does your code introduce? Put an `x` in all the boxes that apply
+-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Chore (non-breaking change which doesn't affect codebase; test, version modification, documentation, etc.)
```

Comment:
The checkboxes would show up as tasklist on the PR.
It is better we ask the user to add the title of the PR with the appropriate tag (bugfix/feat/docs/tests etc).